Rules.md

```diff
@@ -63,6 +63,9 @@ One of:
 1. At the end of a turn, have at least a 10 point advantage over the other team (ends the game)
 1. Get your team to score the most points at the end of turn 10
 
+   **In case of a tie:** The team with the most coins on board wins. If these match, the team
+   with the army which has the highest payment cost, wins. If these match, the allied forces win.
```

Comment:
I would drop the Allied forces tie breaker. There is no shame in sharing the win.